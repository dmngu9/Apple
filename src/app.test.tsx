import * as React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Example from './app';

fdescribe('<Example />', () => {
    test('should render correctly', () => {
        const wrapper = shallow(<Example />);

        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});
