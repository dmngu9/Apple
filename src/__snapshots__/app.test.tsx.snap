// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Example /> should render correctly 1`] = `
<styled.div
  onClick={[Function]}
>
  This is 
  left
</styled.div>
`;
